### ISSUE TITLE
🏁 Loot Rascals ↔ v22.2.0

### ISSUE THREAD

**Issue by shelbos** [itchio/itch-compatibility-watchlist#472]
:rotating_light: game [Loot Rascals](https://hollowponds.itch.io/lootrascals) is broken for me.

:book: Here's the complete [debug log](https://gist.github.com/2cde7796651d69912eb34382d7974ce5).

:running:Its last words were: spawn C:\Users\shelb\AppData\Roaming\itch\apps\Loot Rascals\LootRascals.exe ENOENT

**Comment by fasterthanlime** [comment]
It looks like the executable just disappeared - are you running any antivirus software that may have put it in quarantine?

**Comment by shelbos** [comment]
i don't think so ill double check, i thought id set the antivirus up to let the software run but maybe not. i check and retry. Thanks :)
